docs/requirements.txt

```diff
@@ -25,3 +22,6 @@ huggingface_hub==0.7
 torchaudio==0.11.0
 fasttext==0.9.2
 timm==0.6.5
+torch==1.11.0
+requests==2.28.0
+pathlib==1.0.1
```

Comment:
I think pathlib is already included in python so this isn't needed?  https://pathlib.readthedocs.io/en/pep428/

Reply:
I've removed it